### ISSUE TITLE
Allow users to add custom captions for (YouTube) videos

### ISSUE THREAD

**Issue by MichSpieler** [pressbooks/ideas#531]
### Summary
Let authors add their own captions for embedded videos that do not have complete or adequate captions.

### Problem Description
When authors embed videos that are not WCAG compliant due to captioning issues (missing, auto-generated, or inaccurate), there is currently no way for them to provide visitors an alternative set of captions.

### Affected Users
Authors who embed videos and readers who use assistive technology.

### Importance and Impact
WCAG compliance; if this option is not available, authors would have to remove or replace videos.
Specifically requested by a network manager ([link](https://pressbooks.zendesk.com/agent/tickets/26251) for internal use only).

### Proposed Solution
Offer a alternative media player like [Able Player](https://wordpress.org/plugins/ableplayer)

**Comment by SteelWagstaff** [comment]
Able Player is a great idea. Users can suggest captions to video uploaders or manually add links to their own caption files just beneath embedded videos currently, however.
